src/containers/Application/Dashboard.js

```diff
@@ -31,27 +40,25 @@ const ApplicationDashboardContainer = () => {
   }, [setRelevantDates])
 
   const hackerStatusObject = application.status
-  let hackerStatus =
+  const hackerStatus =
     hackerStatusObject !== undefined &&
     (hackerStatusObject.applicationStatus === 'accepted'
       ? hackerStatusObject.responded
         ? hackerStatusObject.attending
           ? 'acceptedAndAttending'
           : 'acceptedNotAttending'
         : 'acceptedNoResponseYet'
+      : hackerStatusObject.applicationStatus === 'scored'
+      ? 'applied'
       : hackerStatusObject.applicationStatus)
```

Comment:
Nit for code quality: Can you see if you can convert this into if conditions instead of using nested ternary operators? Having too many nested ternary conditions tends to make the code not as maintainable.